mne/io/base.py

```diff
@@ -1881,6 +1928,61 @@ def _convert_slice(sel):
         return sel
 
 
+def _get_scaling(ch_type, target_unit):
+    """Return the scaling factor based on the channel type and a target unit.
+
+    Parameters
+    ----------
+    ch_type: str
+        The channel type.
+    target_unit: str
+        The target unit for the provided channel type.
+
+    Returns
+    -------
+    scaling: int
```

Comment:
This is probably a float in most cases, because the prefixes (like `1e6`) are float types

```suggestion
    scaling : float
```